src/Process/LayeredMaterialNodeProcessing.js

```diff
@@ -227,13 +226,26 @@ export function updateLayeredMaterialNodeImagery(context, layer, node) {
         return;
     }
 
+    const failureParams = node.layerUpdateState[layer.id].failureParams;
     const currentLevel = node.material.getColorLayerLevelById(layer.id);
-    const zoom = node.getCoordsForLayer(layer)[0].zoom || node.level;
-    const targetLevel = chooseNextLevelToFetch(layer.updateStrategy.type, node, zoom, currentLevel, layer);
+    const bestLevel = node.getCoordsForLayer(layer)[0].zoom || node.level;
+    const failedFetchLevel = failureParams ? failureParams.targetLevel : undefined;
+    // if there has already been a failed fetch on previous level
+    // then try at the lower level
+    const targetLevel = failedFetchLevel ? failedFetchLevel - 1 :
```

Comment:
Actually I'm not sure that retrying at `level - 1` makes sense.
Can you give me an example scenario where it would be useful/correct ?

Reply:
As indicated in the description : `For tiles, some wmts level wasn't at their best`

You can try in asia with scanEx example, if you change camera's Range 100000 to 3000 with no animation, the wmts will not be at its best zoom. the reason is in this [comment](https://github.com/iTowns/itowns/pull/618#discussion_r165647342)

Reply:
Isn't this problem caused by the strategy bug returning a targetLevel = 12 while the layer has a max = 10 ? (and you already fixed this).

Reply:
yes that's it.

it also adds the matrix limit case are false.

by example for wmts layer.
it's indicated an interval limit 0 to 15 but the server only returns images from level 0 to 10.
So a level 18 tile can stay at level 0  image because it's blocked by an error at level 15 image

Reply:
Then using `dichotomy` as the default strategy could make more sense than blindly retrying `level - 1` for all failing tiles.

Reply:
yes, it was planned. I'll add it in this PR

Reply:
PR updated, retry use dichotomy